search.py

```diff
@@ -61,10 +61,10 @@ def reindex_app(app):
     machine_class, tasks = SEARCHABLE_APPS[app]
     util.use_random_host('class-%s' % machine_class)
 
-    for task in tasks:
+    for t in tasks:
```

Comment:
You should probably bit a bit more explicit about this, and the next two, parts of the change as it's not as obvious from a brief read that you're avoiding a naming conflict.